uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/apis/leave-request-api.js

```diff
@@ -265,18 +267,18 @@ define([
       /**
        * Calls the addcomment backend API.
        *
-       * @param {String} leaveRequestID - ID of leave request
-       * @param {String} commentText - comment text
-       * @param {String} contactID - ID of contact
+       * @param {string} leaveRequestID - ID of Leave Request
+       * @param {Object} comment - Comment object
        * @param {Object} params
        *
        * @return {Promise}
        */
-      saveComment: function (leaveRequestID, commentText, contactID, params) {
+      saveComment: function (leaveRequestID, comment, params) {
         params = _.assign({}, params, {
           leave_request_id: leaveRequestID,
-          text: commentText,
-          contact_id: contactID
+          text: comment.text,
+          contact_id: comment.contact_id,
+          created_at: moment(comment.created_at, sharedSettings.serverDateFormat).format(sharedSettings.serverDateTimeFormat)
```

Comment:
this will only contain the date and may miss the timestamp or time part of comment? will ` moment(comment.created_at, sharedSettings. serverDateTimeFormat)` help?